src/server/controllers/users.controller.js

```diff
@@ -34,3 +12,15 @@ export function getAll(req, res, next) {
     }
   });
 }
+
+// Get all user list except logined user
+export function getMentorList(req, res, next) {
+  user.find({ 'email' : { $ne : 'session@yoda.com' }},  // if session_auth add, 'session' => req.session.email
+    { email: 0, userId: 0, age: 0, region: 0, skills: 0, regDate: 0 }, (err, doc) => {
```

Comment:
Ok actually 'email: 0' means 'I'll not return email field',
so I'll change this as { userId: 1, email: 0, name: 1, age: 0, field: 1, region: 0, skills: 0, regDate: 0 }...